_ja/functions/unzip3.md

```diff
@@ -0,0 +1,12 @@
+---
+name: unzip3
+---
+
+# `unzip3`
+
+@include [signatures/unzip.md]
+
+`unzip` は、3つのコレクションからなる `Tuple3` を作成します。
```

Comment:
Wrong function name.

Reply:
Fixed in https://github.com/superruzafa/visual-scala-reference/pull/11/commits/5d802388170b18a0bf3b64bcacb31fbcfb97c371